<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zahlenraten</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonRaten">
      <property name="toolTip">
       <string>Findet die Zahl, die genau zwischen den beiden oben definierten Zahlen liegt.</string>
      </property>
      <property name="text">
       <string>Raten</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonTiefer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Finde eine niedrigere Zahl als die aktuell geschätzte.</string>
        </property>
        <property name="text">
         <string>↓ Tiefer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelZahl">
        <property name="text">
         <string>Zahl</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonHoeher">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Finde eine höhere Zahl als die aktuell geschätzte.</string>
        </property>
        <property name="text">
         <string>Höher ↑</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Errate eine Zahl zwischen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>-999999999</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>und</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="minimum">
         <number>-999999999</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
